src/features/rewards/walletPanel/style.ts

```diff
@@ -107,3 +107,21 @@ export const StyledNoticeLink = styled<StyleProps, 'a'>('a')`
   text-decoration: none;
   display: inline-block;
 `
+
+export const StyledSelect = styled<StyleProps, 'select'>('select')`
+  width: 100%;
+  background: inherit;
+  height: 34px;
+  font-size: 14px;
+  border: none;
+  text-align: right;
+  color: #696fdc;
```

Comment:
can we use theme color?

Reply:
We can, I just pulled this value directly from the `Tokens` component

Reply:
let's use theme color, so that it will be easier to do theming